grails-app/assets/javascripts/streama/controllers/dash-ctrl.js

```diff
@@ -158,7 +158,7 @@ angular.module('streama').controller('dashCtrl',
 
     function markCompleted(viewingStatus) {
       alertify.set({buttonReverse: true, labels: {ok: "Yes", cancel: "Cancel"}});
-      alertify.confirm("Are you sure you want to mark this video as completed?", function (confirmed) {
+      alertify.confirm({{'DASHBOARD.MARK_COMPLETED_CONFIRM' | translate}}, function (confirmed) {
```

Comment:
It wont work, unfortunately. The syntax that you have to use is $translate.instant('DASHBOARD.MARK_COMPLETED_CONFIRM') and then you have to inject $translate up in the controller